src/block.js

```diff
@@ -1,12 +1,9 @@
-
 // Copyright (c) 2016-2020 Knuth Project developers.
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-"use strict";
```

Comment:
Do you know why this line was removed?

Reply:
According to the ESLint documentation, ECMAScript modules are always in strict mode, and this strict mode directives in have no effect.